scripts/diagnostics-agent-setup.ps1

```diff
@@ -1,6 +1,7 @@
 Param(
     [Parameter(Mandatory=$true)]
-    [string]$DiagnosticsWindowsBinariesURL
+    [string]$DiagnosticsWindowsBinariesURL,
+    [bool]$IncludeMetrics
```

Comment:
Do not includes these changes in this PR.

Reply:
I will make sure this was not included once the other PR was merged.